universal-login-sdk/lib/api/sdk.ts

```
@@ -140,7 +147,7 @@ class UniversalLoginSDK {
       ...transactionDetails,
       to,
       from: to,
-      data
+      data: data || '0x'
```

should raise an exception if undefined